wix-petri-core/src/test/java/com/wixpress/petri/laboratory/TrackableLaboratoryTest.java

```diff
@@ -90,13 +91,13 @@ public void setUp() throws Exception {
                 with(UserInfoMakers.userId, SOME_USER_GUID));
         registeredUserInfo = aRegisteredUserInfo.make();
         userInfoStorage = new RamUserInfoStorage();
-        testGroupAssignmentTracker = new FakeTestGroupAssignmentTracker();
+        assignmentTrackers = Lists.newArrayList(new FakeTestGroupAssignmentTracker(), new FakeTestGroupAssignmentTracker());
```

Comment:
initially I wanted to comment that the default test setup here should not be changed - just create a list with the one tracker (and add one other test for testing the foreach).

but this got me thinking that maybe it would be best to not touch the core module at all and just have LaboratoryFilter create a tracker which internally has the BITracker and the new AmplitudeTracker - this way the core functionality doesnt change and you only need 1 unit test for it.

wdyt?